boards/swervolf/Makefile

```diff
@@ -0,0 +1,20 @@
+# Makefile for building the tock kernel for the SweRVolf platform
+
+TARGET=riscv32imc-unknown-none-elf
+PLATFORM=swervolf
+
+include ../Makefile.common
+
+# Default target for installing the kernel.
+.PHONY: install
+install: sim
+
+setup-sim:
+	fusesoc library add fusesoc-cores https://github.com/fusesoc/fusesoc-cores
+	fusesoc library add swervolf https://github.com/chipsalliance/Cores-SweRVolf
+	wget https://raw.githubusercontent.com/chipsalliance/Cores-SweRVolf/master/sw/makehex.py
```

Comment:
Can you add a notice that this downloads stuff from the internet _and_ add a checksum verification here? I wouldn't want to have anything downloaded from another repository as part of the Makefile Infrastructure, even worse it being executable code.

Reply:
The file is tiny so I just checked it in.